app/assets/stylesheets/layout_components/forms.scss

```diff
@@ -159,8 +155,8 @@ input[type="checkbox"] + label,
   cursor: pointer;
   padding: 0 $base-padding/2 0 $base-padding/2;
 
-  &:before{
-    content: ' ';
+  &::before {
+    content: " ";
```

Comment:
really minor, but this is inconsistent with `content: " ";` below on line 212

Reply:
there is actually a subtle difference here, in this instance it is saying put a space before the content, whereas the one below is saying don't put a space, so they don't need to match

Reply:
Ah, makes sense.